mmdet/models/roi_heads/point_rend_roi_head.py

```diff
@@ -207,3 +208,153 @@ def aug_test_mask(self, feats, img_metas, det_bboxes, det_labels):
+            refined_mask_pred = F.interpolate(
+                refined_mask_pred,
+                scale_factor=self.test_cfg.scale_factor,
+                mode='bilinear',
+                align_corners=False)
+            # If `subdivision_num_points` is larger or equal to the
+            # resolution of the next step, then we can skip this step
+            num_rois, channels, mask_height, mask_width = \
+                refined_mask_pred.shape
+            if (self.test_cfg.subdivision_num_points >=
+                    self.test_cfg.scale_factor**2 * mask_height * mask_width
+                    and
+                    subdivision_step < self.test_cfg.subdivision_steps - 1):
+                continue
+            point_indices, rel_roi_points = \
+                self.point_head.get_roi_rel_points_test(
+                    refined_mask_pred, label_pred, cfg=self.test_cfg)
+            fine_grained_point_feats = self._onnx_get_fine_grained_point_feats(
+                x, rois, rel_roi_points)
+            coarse_point_feats = point_sample(mask_pred, rel_roi_points)
+            mask_point_pred = self.point_head(fine_grained_point_feats,
+                                              coarse_point_feats)
+
+            point_indices = point_indices.unsqueeze(1).expand(-1, channels, -1)
+            refined_mask_pred = refined_mask_pred.reshape(
+                num_rois, channels, mask_height * mask_width)
+
+            is_trt_backend = os.environ.get('ONNX_BACKEND') == 'MMCVTensorRT'
+            # avoid ScatterElements op in ONNX for TensorRT
+            if is_trt_backend:
+                mask_shape = refined_mask_pred.shape
+                point_shape = point_indices.shape
+                inds_dim0 = torch.arange(point_shape[0]).reshape(
+                    point_shape[0], 1, 1).expand_as(point_indices)
+                inds_dim1 = torch.arange(point_shape[1]).reshape(
+                    1, point_shape[1], 1).expand_as(point_indices)
+                inds_1d = inds_dim0.reshape(
+                    -1) * mask_shape[1] * mask_shape[2] + inds_dim1.reshape(
+                        -1) * mask_shape[2] + point_indices.reshape(-1)
+                refined_mask_pred = refined_mask_pred.reshape(-1)
+                refined_mask_pred[inds_1d] = mask_point_pred.reshape(-1)
+                refined_mask_pred = refined_mask_pred.reshape(*mask_shape)
+            else:
+                refined_mask_pred = refined_mask_pred.scatter_(
+                    2, point_indices, mask_point_pred)
+
+            refined_mask_pred = refined_mask_pred.view(num_rois, channels,
+                                                       mask_height, mask_width)
+
+        return refined_mask_pred
+
+    def mask_onnx_export(self,
+                         x,
+                         img_metas,
+                         det_bboxes,
+                         det_labels,
+                         rescale=False,
```

Comment:
Seems `rescale` would never be used when doing the `onnx_export`, you may delete all the `rescale` related codes. But it is best to confirm by  @RunningLeon